shell/platform/android/io/flutter/plugin/common/MethodChannel.java

```diff
@@ -173,8 +173,10 @@ public interface MethodCallHandler {
      * <p>Any uncaught exception thrown by this method will be caught by the channel implementation
      * and logged, and an error result will be sent back to Flutter.
      *
-     * <p>The handler is called on the platform thread (Android main thread). For more details see
-     * <a href="https://github.com/flutter/engine/wiki/Threading-in-the-Flutter-Engine">Threading in
+     * <p>The handler is called on the platform thread (Android main thread) by default, or
+     * otherwise on the thread specified by the task queue provided to the associated {@link
+     * MethodChannel} when it was created. See also <a
+     * href="https://github.com/flutter/engine/wiki/Threading-in-the-Flutter-Engine">Threading in
```

Comment:
This link is dead. The updated link is https://github.com/flutter/flutter/wiki/The-Engine-architecture#threading.

Reply:
Thanks, this has been corrected